agent/tool-scripts/base-tool

```diff
@@ -579,8 +579,10 @@ install)
 	function check_required_rpm {
 		local _tpn="${1}"
 		local _tpv="${2}"
-		local _installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
-		local rc=${?}
+		local _installed_rpm
+		local rc
+    _installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
+    rc=${?} 
```

Comment:
These lines should be indented to align with the other function statements.

Reply:
I couldn't see any indentation issues in my `vim`, Maybe due to some other issues. will make the change :)

Reply:
The adjacent lines appear to be using (two) hard-tab characters for indentation, while these lines are using (4) blank space characters.

The file does have the
```
# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; tab-width: 8 -*-
```
annotation which supports the tab-based indentation.

Reply:
Thank you. I will take a look :)

Reply:
FWIW, I pulled this change out into its own PR keeping authorship as @riya-17, see https://github.com/distributed-system-analysis/pbench/pull/2723.